pkgs/os-specific/linux/minimal-bootstrap/default.nix

```diff
@@ -4,6 +4,7 @@
 , hostPlatform
 , fetchurl
 , checkMeta
+, make-minimal-bootstrap-sources
 }:
 
 lib.makeScope
```

Comment:
Add `make-minimal-bootstrap-sources` to the scope inputs here so it doesn't need to be manually passed through?

Reply:
5dacb6894093

Reply:
Sorry should clarify that this line would be the best place to declare it :) https://github.com/NixOS/nixpkgs/blob/3b5e8694a52cdcc07a32f324ebe5f0ddbd14fe33/pkgs/os-specific/linux/minimal-bootstrap/default.nix#L14

Reply:
3d3fe75cbc6f